minecraft/old-combat-system.md

```diff
@@ -11,4 +11,4 @@ Du solltest keinen Minecraft 1.8(.8) Server verwenden aus mehreren Gründen.
 
 - <https://www.spigotmc.org/resources/protocolsupportstuff.44527/>
 
- <https://www.spigotmc.org/resources/oldcombatmechanics-disable-1-9-hit-cooldown.19510/>
+- <https://www.spigotmc.org/resources/oldcombatmechanics-disable-1-9-hit-cooldown.19510/>
```

Comment:
The formatting of the plugin links feels odd. The point in line 9 mentions that some plugins can restore the old combat system, but the plugins are at the same hierarchy than the message.

I would suggest to indent all 3 plugins links with 2 spaces to make them a subpoint of the last one.